test/lazy/test_broadcasted_lazy_tensor.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import torch
+import unittest
+from gpytorch import lazify
+from gpytorch.test.lazy_tensor_test_case import LazyTensorTestCase
+
+class TestSumLazyTensorBroadcasting(unittest.TestCase):
```

Comment:
It would be better if these tests were part of `test_sum_lazy_tensor` and used the LazyTensorTestCase.

Reply:
The tests are moved into `test_sum_lazy_tensor` now but given that they aren't dealing with symmetric matrices necessarily I'm not sure if they ought to use LazyTensorTestCase.